posthog/tasks/usage_report.py

```diff
@@ -326,10 +326,20 @@ def get_teams_with_event_count_lifetime() -> List[Tuple[int, int]]:
 
 
 @timed_log()
-def get_teams_with_billable_event_count_in_period(begin: datetime, end: datetime) -> List[Tuple[int, int]]:
+def get_teams_with_billable_event_count_in_period(
+    begin: datetime, end: datetime, count_distinct: bool = False
+) -> List[Tuple[int, int]]:
+    # count only unique events
+    if count_distinct:
+        # Uses the same expression as the one used to de-duplicate events on the merge tree:
+        # https://github.com/PostHog/posthog/blob/master/posthog/models/event/sql.py#L92
+        distinct_expression = "distinct toDate(timestamp), event, cityHash64(distinct_id), cityHash64(uuid)"
+    else:
+        distinct_expression = "1"
+
     result = sync_execute(
```

Comment:
could we catch and either log or submit some metric if it OOMed <- that's quite important as it means we might not bill someone at all

Reply:
should be covered with regular handling